src/Radicle/Internal/Pretty.hs

```diff
@@ -64,6 +64,7 @@ instance forall t. (Copointed t, Ann.Annotation t) => PrettyV (Ann.Annotated t V
           v' :<| Empty -> brackets $ prettyV v'
           _            -> brackets . sep $ prettyV <$> toList vs
         PrimFn i -> pretty i
+        Macro _ -> angles "macro"
```

Comment:
We can't just show it?

Reply:
Oh I have some comments about this, see IRC.